src/components/MessagesList/MessagesGroup/MessagesGroup.spec.js

```diff
@@ -1,27 +1,34 @@
 import { createLocalVue, shallowMount } from '@vue/test-utils'
 import { cloneDeep } from 'lodash'
+import { createPinia, setActivePinia } from 'pinia'
 import Vuex from 'vuex'
 
 import MessagesGroup from './MessagesGroup.vue'
 import MessagesSystemGroup from './MessagesSystemGroup.vue'
 
 import { ATTENDEE } from '../../../constants.js'
 import storeConfig from '../../../store/storeConfig.js'
+import { useGuestNameStore } from '../../../stores/guestNameStore.js'
 
 describe('MessagesGroup.vue', () => {
 	const TOKEN = 'XXTOKENXX'
 	let store
 	let localVue
 	let testStoreConfig
 	let getGuestNameMock
+	let pinia
 
 	beforeEach(() => {
 		localVue = createLocalVue()
 		localVue.use(Vuex)
+		pinia = createPinia()
+		setActivePinia(pinia)
+
+		const guestNameStore = useGuestNameStore()
 
 		testStoreConfig = cloneDeep(storeConfig)
 		getGuestNameMock = jest.fn()
-		testStoreConfig.modules.guestNameStore.getters.getGuestName = () => getGuestNameMock
+		guestNameStore.getGuestName = getGuestNameMock
```

Comment:
You added a store here, so you can directly use it for test `renders guest display name`:
```js
	// Arrange
	guestNameStore.addGuestName({
		token: TOKEN,
		actorId: 'actor-1',
		actorDisplayName: 'guest-one-display-name',
	})
```

And remove mock from the file completely - we've tested it in store test already:
```js
	// Assert
-	expect(guestNameStore.getGuestName).toHaveBeenCalledWith(TOKEN, 'actor-1')
```